lightning/src/ln/channelmanager.rs

```diff
@@ -4487,6 +4523,13 @@ where
 					peer_state.outbound_v1_channel_by_id.retain(|chan_id, chan| process_unfunded_channel_tick(chan_id, &mut chan.context, &mut chan.unfunded_context));
 					peer_state.inbound_v1_channel_by_id.retain(|chan_id, chan| process_unfunded_channel_tick(chan_id, &mut chan.context, &mut chan.unfunded_context));
 
+					for (chan_id, req) in peer_state.inbound_channel_request_by_id.iter_mut() {
+						if { req.ticks_remaining -= 1 ; req.ticks_remaining } <= 0 {
```

Comment:
Ope, hmm... after addressing the issue below I think that it might make more sense to have a separate loop. In particular, the `retain` lambda gets sad about the use of `peer_state`.